concepts/05 Widgets/DateBox/00 Getting Started with DateBox/00 Getting Started with DateBox.md

```diff
@@ -0,0 +1,244 @@
+                if (date.getDate() === holiday.getDate() && date.getMonth() === holiday.getMonth()) {
+                    return true;
+                }          
+            }
+            return false;
+        }
+    });
+
+    <!-- tab: index.html -->
+    <!DOCTYPE html>
+    <html>
+        <head>
+            <script type="text/javascript" src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+            <link rel="stylesheet" href="https://cdn3.devexpress.com/jslib/20.1.4/css/dx.common.css">
+            <link rel="stylesheet" href="https://cdn3.devexpress.com/jslib/20.1.4/css/dx.light.css">
+            <script type="text/javascript" src="https://cdn3.devexpress.com/jslib/20.1.4/js/dx.all.js"></script>
+            <script type="text/javascript" src="index.js"></script>
+        </head>
+        <body class="dx-viewport">
+            <div id="dateBox"></div>
+        </body>
+    </html>
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-date-box    
+        type="datetime"
+        [min]="minDate"
+        [max]="now"
+        [value]="now"
+        (onValueChanged)="onValueChanged($event)"
+        [disabledDates]="getDisabledDates"
+    >
+    </dx-date-box>
+
+    <!-- tab: app.component.ts -->
+    import { Component } from '@angular/core';
+
+    @Component({
+        selector: 'app-root',
+        templateUrl: './app.component.html',
+        styleUrls: ['./app.component.css']
+    })
+    export class AppComponent {
+        title = 'Getting started with DateBox';
+        holidays: Date[];
+        now: Date = new Date();
+        minDate: Date = new Date(1900, 0, 1);
+
+        constructor(service: AppService){
+            this.holidays = service.getHolidays();
+            this.getDisabledDates = this.getDisabledDates.bind(this);
+        }
+
+        onValueChanged(e){
+            console.log(e.previousValue);
+            console.log(e.value);
+        }
+
+        getDisabledDates(args): boolean {
+            const dayOfWeek = args.date.getDay();
+            const isWeekend = args.view === "month" && (dayOfWeek === 0 || dayOfWeek === 6 );
+            return isWeekend || this.isHoliday(args.date);
+        }
+
+        isHoliday(date): boolean {
+            for (let holiday of this.holidays) {
+                if (date.getDate() === holiday.getDate() && date.getMonth() === holiday.getMonth()) {
+                    return true;
+                }          
+            }
+            return false;
+        }
+    }
+
+    <!-- tab: app.module.ts -->
+    import { BrowserModule } from '@angular/platform-browser';
+    import { NgModule } from '@angular/core';
+    import { AppComponent } from './app.component';
+
+    import { DxDateBoxModule } from 'devextreme-angular';
+
+    @NgModule({
+        declarations: [
+            AppComponent
+        ],
+        imports: [
+            BrowserModule,
+            DxDateBoxModule
+        ],
+        providers: [ ],
```

Comment:
I think the data service providing holidays should be imported and registered in the module.

Reply:
The service imported in app.component.ts and not in the app.module.ts should suffice. See:
https://js.devexpress.com/Documentation/Guide/Widgets/ButtonGroup/Getting_Started_with_ButtonGroup/
https://js.devexpress.com/Demos/WidgetsGallery/Demo/SelectBox/SearchAndEditing/Angular/Light/